internal/pkg/manifest/env.go

```diff
@@ -127,14 +198,28 @@ type environmentObservability struct {
 }
 
 // IsEmpty returns true if there is no configuration to the environment's observability.
-func (o environmentObservability) IsEmpty() bool {
-	return o.ContainerInsights == nil
+func (o *environmentObservability) IsEmpty() bool {
+	return o == nil || o.ContainerInsights == nil
+}
+
+func (o *environmentObservability) loadConfig(tele *config.Telemetry) {
+	if tele == nil {
+		return
+	}
+	o.ContainerInsights = &tele.EnableContainerInsights
 }
 
 type environmentHTTPConfig struct {
 	Public publicHTTPConfig `yaml:"public,omitempty"`
 }
 
+func (cfg *environmentHTTPConfig) loadLBConfig(env *config.CustomizeEnv) {
+	if env.IsEmpty() {
+		return
+	}
+	cfg.Public.Certificates = env.ImportCertARNs
```

Comment:
What about the internal ALB certs? Will it have a different name in SSM?

Reply:
Yeah, once @huanjani adds it to SSM, we will have to update this method.